### PR TITLE
Converted to swift 4.2

### PR DESCRIPTION
**Pull request by wytzen**

### CODE REVIEW

**Review by martijnkooijman — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -59,29 +57,10 @@ let notification = BSForegroundNotification(titleLabel: "title", subtitleLabel:
 notification.timeToDismissNotification = 10 //4 is by default
 ```
 
-######Set delegate which conform to protocol `BSForegroundNotificationDelegate`:
```
Comment (dennis-montana) [inline README.md:62]: Why did you remove this part of the readme?

File: Example/Pods/Target Support Files/BSForegroundNotification/Info.plist [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
-  <key>CFBundleVersion</key>
-  <string>${CURRENT_PROJECT_VERSION}</string>
-  <key>NSPrincipalClass</key>
-  <string></string>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleExecutable</key>
+	<string>${EXECUTABLE_NAME}</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```
Comment (dennis-montana) [inline Example/Pods/Target Support Files/BSForegroundNotification/Info.plist]: Are you sure this is needed?

File: README.md (hunk 1 above)
Comment (wytzen) [reply README.md:62]: In the current version it is not possible to add a delegate anymore. One should pass a callback to the presentNotification

**Review by dennis-montana — APPROVED** [review]
